src/twisted/mail/pop3.py

```diff
@@ -855,7 +858,13 @@ def do_PASS(self, password):
 
         @type password: L{bytes}
         @param password: A password.
+
+        @type args: L{tuple} or L{None}
+        @param args: Other words composing the password.
         """
+        if args:
+            password += ' ' + ' '.join(args)
```

Comment:
This brings me down a rabbit hole: doesn't that mean stuff like `return self.processCommand(*line.split(' '))` in line 565 should use `b' '` as well? On a quick check, I see this overall in `pop3.py`.
If that's the case, am I correct to assume we should open a new ticket and fix it there?